source/blog/2017-10-27-webadmin-lldp.html.md

```diff
@@ -0,0 +1,72 @@
+---
+title: LLDP information in webadmin
+author: amusil
+tags: webadmin, networking
+date: 2017-10-27 09:00:00 CET
+---
+
+In oVirt 4.2 we have introduced support for LLDP protocol. 
+LLDP stands for Link Layer Discovery Protocol, which is used by network devices 
+for advertising identity and capabilities to neighbors on a LAN. 
+[More information about LLDP](https://learningnetwork.cisco.com/docs/DOC-26851).
+
+READMORE
+
+## Useful information
+
+The information gathered by the protocol can be used for better network 
```

Comment:
Again, this blog post lacks context:
1. What are you trying to achieve in this post?
2. What do you want the reader to get out of reading this post? (use this feature, complete a certain action)?

In short, a good post should inform the reader of the "What, Why, and How"
1. A descriptive title that tells the reader what to expect from reading this post
2. An introduction: e.g. "This post will show you how to achieve A. B. C. First, a brief introduction."
3. (if the purpose is to show a user how to install / use a feature) Clearly defined steps.
4. Some kind of conclusion.